Makefile

```diff
@@ -23,14 +23,31 @@ RM=--rm
 RUN_FLAGS=-ti
 BUILD_IN_CONTAINER=true
 GO_ENV=GOGC=off
-GO=env $(GO_ENV) go
-NO_CROSS_COMP=unset GOOS GOARCH
-GO_HOST=$(NO_CROSS_COMP); $(GO)
-WITH_GO_HOST_ENV=$(NO_CROSS_COMP); $(GO_ENV)
+GO_ENV_ARM=$(GO_ENV) CC=/usr/bin/arm-linux-gnueabihf-gcc
```

Comment:
Yes.

Since whenever we call the go binary we use `env $(GO_ENV) go`, the `CC` variable won't be picked up.

We can either leave it this way or do:

```
ifeq ($(GOARCH),arm)
CC=/usr/bin/arm-linux-gnueabihf-gcc
GO=env CC=$(CC) $(GO_ENV_ARM) go
...
```